ckan/common.py

```diff
@@ -208,3 +208,41 @@ def _get_session():
 # Provide a `c`  alias for `g` for backwards compatibility
 g = c = LocalProxy(_get_c)
 session = LocalProxy(_get_session)
+
+# replacement for converters that previously came from paste.deploy.converters
+# [#4797]
```

Comment:
We can get this information from the commit message, so no need to add a comment to source code